e2e/card-view-component.e2e.ts

```diff
@@ -0,0 +1,103 @@
+/*!
+ * @license
+ * Copyright 2016 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ import LoginPage = require('./pages/adf/loginPage');
+ import NavigationBarPage = require('./pages/adf/navigationBarPage');
+ import TestConfig = require('./test.config');
+ import resources = require('./util/resources');
+ import AlfrescoApi = require('alfresco-js-api-node');
+ import { UsersActions } from './actions/users.actions';
+ import { AppsActions } from './actions/APS/apps.actions';
+ import CardViewPageComponent = require('./pages/adf/cardViewPageComponent');
+ import Util = require('./util/util');
+
+ describe('A suite', () => {
```

Comment:
This name is too generic can you please use the name of the component that are you testing?